### ISSUE TITLE
Accuracy of pretrained model ?

### ISSUE THREAD

**Issue by EmonLu** [dvlab-research/Simple-SR#24]
Hi, 
I tried to test the PSNR/SSIM of your pre-trained model but got results different from the paper.
I've got no idea what was going wrong. Did not change any parameters.
![image](https://user-images.githubusercontent.com/53183584/206127107-d0713cce-6ea3-44cc-93de-46e2e9b250a3.png)

**Comment by fenglinglwb** [comment]
It doesn't make sense that the PSNR/SSIM of LAPAR_C > those of LAPAR_A. Maybe you should check the LR/HR data and evaluation scripts you used.

**Comment by EmonLu** [comment]
The LR are downscaled from HR with BICUBIC(cv2.resize). Does it not make sense?
And I used test_sample.py in the repo.
